logs_api/lib/opentelemetry/logs/logger_provider.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Logs
+    # No-op implementation of a logger provider
+    class LoggerProvider
+      # Returns a {Logger} instance.
+      #
```

Comment:
The specification requires some kind of documentation that makes it clear that the log bridge API is not meant for direct consumption by application authors.

See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#artifact-naming

> The Logs Bridge API is not intended to be called by application developers directly, and SHOULD include documentation that discourages direct use.

Here's an example of how Java meets this requirement:

https://github.com/open-telemetry/opentelemetry-java/blob/90ab3665dc0fb56598fcf9b9c15941eedb3822f8/api/all/src/main/java/io/opentelemetry/api/logs/LoggerProvider.java#L14-L16

Reply:
Thank you! I'll update the documentation.

Reply:
I've added a statement discouraging direct use by app developers in a006fba84e7c69d869403860e705b3184bdb9e17

Like Java, it's included in the Logger Provider and the Logger class definitions. Let me know if you think it should be included in all logger-related classes. I couldn't find similar notes in [PHP](https://github.com/open-telemetry/opentelemetry-php/blob/main/src/API/Logs/LoggerProviderInterface.php), [Python](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py), or [JavaScript](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/api-logs/src/types/LoggerProvider.ts).

Reply:
Maybe it should also be included in the definition of the `OpenTelemetry::Logs` module?